### ISSUE TITLE
Segfault for specific heading omputing

### ISSUE THREAD

**Issue by pzia** [v-l-m/vlm#613]
**Reported by paparazzia on 17 Jun 2011 18:01 UTC**
Jun 17 10:50:03 srv-qc210-01 kernel: [php[7780](7464787.600325]): segfault at 7f994fca8574 ip 00007f9d509ce1a6 sp 00007fffb1fd37d0 error 4 in vlmc.so[7f9d5098e000+60000]
 le dbut des emm... :)
 vu
 (224) Checking user 17143: (race 20110601) 
      Prev Hdg: 6.00002 PIM=5/Heading=6.0000173297123, Reaching position=089.59'59"N/000.00'00"W MyWP=(89.999722222222,0) NO WPH , dist=147.091nm
         Position update (WSpd=6.3, WHdg=322, Hours=0.0833, ANG=43.25, Hdg=6.0000173297123, BS=5.922, Dist=0.49354079503804nm) ... done
         Checking for coast crossing (vlm-c)
         Using: /home/vlm/vlmdatas/gshhs/gshhs_f.b

```
     Next Waypoint is 1, checking for WP crossing... 
             * WP   : 71.000000, 48.000000 <---> 71.000000, 49.000000
             * BOAT : 87.548199, -0.089925 <---> 87.556379, -0.069792:        *** NO ***
             *** NWP=1, DNM=698.102171 
             *** ranking updated.
     ** Pilotmode=5
```

 (END)

**Comment by pzia** [comment]
**Commented by paparazzia on 18 Jun 2011 13:11 UTC**
JDD pour reproduire le pb :

```
REPLACE positions values (%last_timestamp%, -69.792375816942, 87556.379228668, %idusers% , %idraces% );
```

```

UPDATE users SET engaged = %idraces%, targetlat = 89.999722222222, targetlong =0, pilotmode= 3 where idusers = 10548;
```

**Comment by pzia** [comment]
**Commented by paparazzia on 18 Jun 2011 13:16 UTC**
Et a donne dans les logs :

```

Jun 18 13:14:08 testing kernel: [php[7490](7136060.588272]): segfault at d20 ip 00007f74836cb1a5 sp 00007fff50f1a5f0 error 6 in vlmc.so[7f7483687000+60000]
```

**Comment by pzia** [comment]
**Commented by spf on 18 Jun 2011 14:26 UTC**
(In [2670]) more checking (Fixes #613)

**Comment by pzia** [comment]
**Commented by spf on 18 Jun 2011 14:27 UTC**
(In [backport of [2670](2671])) (See #613)
